java/memory/src/main/java/org/apache/arrow/memory/BaseAllocator.java

```diff
@@ -465,20 +464,14 @@ public synchronized void close() {
         }
 
       }
-    } else {
-      if (!childAllocators.isEmpty()) {
-        outstandingChildAllocators.append("Outstanding child allocators : \n");
-        synchronized (childAllocators) {
-          for (final BaseAllocator childAllocator : childAllocators.keySet()) {
-            outstandingChildAllocators.append(String.format("  %s", childAllocator.toString()));
-          }
-        }
-      }
```

Comment:
this is useful information. can you please add this to the exception msg below ?